packages/account/src/wallet/wallet-unlocked.test.ts

```diff
@@ -26,7 +26,7 @@ describe('WalletUnlocked', () => {
   const expectedAddress = '0xf1e92c42b90934aa6372e30bc568a326f6e66a1a0288595e6e3fbd392a4f3e6e';
   const expectedMessage = 'my message';
   const expectedSignedMessage =
-    '0x8eeb238db1adea4152644f1cd827b552dfa9ab3f4939718bb45ca476d167c6512a656f4d4c7356bfb9561b14448c230c6e7e4bd781df5ee9e5999faa6495163d';
+    '0x9a16a5452ae96c13732716682600a6054661ba43994c9c335a4655caf31e68f0b1ec2e7c7ffe304a1b9d30734e24bb8ef3a5076edbc4510f0a3de3d9dcba9667';
 
   it('Instantiate a new wallet', async () => {
```

Comment:
we should ensure the current method does not work; a break change here can be catastrophic.

Like bako predicate, which uses the current sign message to verify ownership,